tracee-rules/main.go

```diff
@@ -113,6 +113,10 @@ func main() {
 				Name:  "rules-dir",
 				Usage: "directory where to search for rules in OPA (.rego) or Go plugin (.so) formats",
 			},
+			&cli.BoolFlag{
+				Name:  "partial-eval",
```

Comment:
Maybe we can group flags related to OPA / Rego with the common prefix `opa-`
```suggestion
				Name:  "opa-partial-eval",
```

Reply:
Roger that - I prefixed `rego`: https://github.com/aquasecurity/tracee/pull/979/commits/e7583199bd004ae051d0342e20bfdaee8c87f656